requirements.txt

```diff
@@ -1,7 +1,16 @@
-beautifulsoup4
-aiohttp
-async_timeout>=3.0,<4.0
-urllib3
-requests
-lxml
-tqdm
+aiohttp==3.7.4.post0
```

Comment:
There are dependencies that were needed such as `click` that was not declared in this file. I presume it has always worked for you guys since you already have them pre-installed in your machines.

The issue will hit someone like me, who comes in with a clean environment with nothing pre-installed.

Reply:
I think we should keep it as general as possible. If you can find what version breaks, then we can change the requirements.

Reply:
Got it. Change reverted.